draft-ietf-webpush-protocol.xml

```diff
@@ -938,6 +938,11 @@ Host: push.example.net
           then the push service MUST return a 410 (Gone) response to the application server
           monitoring the receipt subscription.
         </t>
+        <t><list style="hanging"><t hangText="Note:">
+          An application server also receives the push message resource URI when it sends
+          a push message.  This allows an application server to acknowledge messages it

```

Comment:
The PS ACK would need to be called out. What scenario is motivating this proposal  now? I'm also thinking that if the Application Server is deleting an inflight-message then it might want to delete one or more in-flight messages.